src/styles/abstracts/_tokens.scss

```diff
@@ -1,6 +1,6 @@
 /* outlines */
 $outline-color: var(--color-accent-500);
-$outline-focus: 2px solid var(--outline-color);
+$outline-focus: 2px solid $outline-color;
```

Comment:
I guess here is huge nesting (css variable > scss variable > another scss variable). Please use `$outline-focus: 2px solid var(--color-accent-500)`;

Reply:
fix